samples/bookinfo/src/reviews/reviews-wlpcfg/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM websphere-liberty:latest
+FROM websphere-liberty:19.0.0.4-javaee8
```

Comment:
@johnma14 why are you locking down versions here instead of installing the latest?  I worry that this will have the opposite effect of your intention - we will fall further behind on our dependencies which will decrease our security

/hold

Reply:
@duderino In the following PR: https://github.com/istio/istio/pull/13774 @frankbu mentioned it was not a good idea to use `latest` tag. I think we need to a consensus on whether to use  `latest` tag or a specific version

Reply:
using latest for any container creates headaches for compliance.  All dockerfiles throughout our repos should use specific versions, otherwise they fail PCI complaince.